scripts/globals/mobs.lua

```diff
@@ -59,14 +59,16 @@ function phOnDespawn(ph,phList,chance,cooldown)
         local nm = GetMobByID(nmId);
         if (nm ~= nil) then
             local pop = nm:getLocalVar("pop");
-            if (os.time() > pop and not lotteryPrimed(phList) and math.random(100) <= chance) then
+            local swapped = nm:getLocalVar("swapped");
```

Comment:
what's the point of this swapped localVar? the lotteryPrimed() function already tells us if a lottery NM is already set to spawn in place of its PH.

Reply:
oh i see, it's done via nm:getRespawnTime
@wrenffxi totally didn't see it, thanx